proto/storage/compliance_operator_v2.proto

```diff
@@ -10,13 +10,36 @@ import "storage/user.proto";
 
 package storage;
 
+// Represents the role of the node within the cluster
+enum NodeRole {
+    INFRA = 0;
+    WORKER = 1;
+    MASTER = 2;
+}
+
+// Represents the scan type whether a node or platform scan.
+enum ScanType {
+    UNSET_SCAN_TYPE = 0;
+    NODE_SCAN = 1;
+    PLATFORM_SCAN = 2;
+}
+
+message Profile {
+    string                    profile_id               = 1 [(gogoproto.moretags) = 'search:"-" sql:"fk(ComplianceOperatorProfileV2:id),no-fk-constraint"'];
+}
+
+message ClusterScanStatus {
+    string                    cluster_id = 1 [(gogoproto.moretags) = 'search:"Cluster ID,hidden,store" sql:"fk(Cluster:id),no-fk-constraint,type(uuid)"'];
```

Comment:
Is this to be used to represent overall error or per profile or can be used for both?

Reply:
The idea was per cluster.  For example we apply a schedule to 6 clusters but sensor has trouble applying the CRs with one.  We need to be able to see that.  It only applies to ScanSettings at the moment.  So I'm going to scope it to ScanSettings so it is more clear.  I wouldn't think we would need an error per profile on settings.